tool/clean/ec2_instance_cleanup/cleaner.py

```diff
@@ -0,0 +1,139 @@
+import json
+import os
+import logging
+
+import boto3
+from _datetime import datetime, timedelta, timezone
+from typing import List
+
+INSTANCE_STATE_RUNNING = 'running'
+EKS_CLUSTER_SECURITY_GROUP_PREFIX = 'eks-cluster-sg-'
+K8S_INSTANCE_NAME_PREFIX = 'k8s-'
+K8S_KEYNAME_PREFIX = 'k8s'
```

Comment:
nit: Is there a reason we can't do this? Does `KEYNAME` not start with a `-` as well?
```suggestion
K8S_RESOURCE_NAME_PREFIX = 'k8s-'
```

Reply:
Did a quick spotcheck and yes, the KEYNAMEs start with `k8s-`. I will update this.